crates/astria-sequencer-relayer/src/config/mod.rs

```diff
@@ -84,7 +87,11 @@ impl Default for Config {
             gas_limit: crate::data_availability::DEFAULT_PFD_GAS_LIMIT,
             disable_writing: false,
             block_time: DEFAULT_BLOCK_TIME,
-            validator_key_file: DEFAULT_VALIDATOR_KEY_FILE.into(),
+            validator_key_file: Path::new(&home::home_dir().unwrap())
```

Comment:
Don't use homedirs in services. Just have the service open the path and fail if it can't find it.